11-node-frameworks/classworks/Agatha Rudko/index.js

```diff
@@ -0,0 +1,58 @@
+const serve = require('koa-static');
+const Koa = require('koa');
+const session = require('koa-generic-session');
+const bodyParser = require('koa-bodyparser');
+const Router = require('@koa/router');
+const joi = require('joi');
+const router = new Router();
+
+const app = new Koa();
+
+
+const schem = joi.object({
+  name: joi.string().required(),
+  surname: joi.string().min(3),
+});
+
+function get(ctx) {
```

Comment:
You should count views for every request to the server. Tip: use middleware, pay attention to how body-parser works